modules/ca/output.tf

```diff
@@ -2,10 +2,22 @@ output "bucket" {
   value = aws_s3_bucket.ca_application_data.bucket
 }
 
+output "bucket_arn" {
```

Comment:
If we are adding arn we might as well add id just so we don't have to in future... maybe 🤷‍♂️

Reply:
I could add that as well. I was going for outputs that I was immediately using in the `nerves_hub_infrastructure` repo but extra outputs can't hurt.